Ginger/GingerCoreCommon/VariablesLib/VariableSelectionList.cs

```diff
@@ -55,6 +58,24 @@ public string OptionalValues
 
         public string SelectedValue { set { Value = value; OnPropertyChanged(nameof(SelectedValue)); } get { return Value; } }
 
+        private bool mRandomOrder;
+        [IsSerializedForLocalRepository]
+        public bool RandomOrder
```

Comment:
what "RandomOrder" used for?

Reply:
that was part of the list of variable functionalities, but it is not required here so I deleted it